sdk/tests/table.rs

```diff
@@ -218,34 +218,34 @@ async fn define_foreign_table_group(cond: bool, agr: &str) -> Result<(), Error>
 	};
 	let sql = format!(
 		"
-		UPDATE wallet:1 CONTENT {{ value: 20.0, day: 1 }} RETURN NONE;
-		UPDATE wallet:2 CONTENT {{ value: 5.0, day: 1 }} RETURN NONE;
+		UPSERT wallet:1 CONTENT {{ value: 20.0dec, day: 1 }} RETURN NONE;
```

Comment:
I'm curious whether these changes were necessary to make the tests pass or you just added them to be explicit about their types?

Reply:
I can't remember right now @rushmorem . Might be worth reverting them and seeing. That was done a while ago!